cmd/antrea-agent/agent.go

```diff
@@ -880,7 +880,7 @@ func run(o *Options) error {
 			klog.InfoS("Waiting for AntreaProxy to be ready")
 			if err := wait.PollUntilContextCancel(wait.ContextForChannel(stopCh), time.Second, false, func(ctx context.Context) (bool, error) {
 				klog.V(2).InfoS("Checking if AntreaProxy is ready")
-				return proxier.GetProxyProvider().SyncedOnce(), nil
+				return proxyServer.GetProxier().GetProxyProvider().SyncedOnce(), nil
```

Comment:
I think I commented this on your other PR: it would be good to simplify this to either `proxyServer.SyncedOnce()` or ` proxyServer.GetProxier().SyncedOnce()`, whichever one makes more sense.